app/screens/send-bitcoin-screen/scanning-qrcode-screen.tsx

```diff
@@ -190,19 +190,32 @@ export const ScanningQRCodeScreen: React.FC<ScanningQRCodeScreenProps> = ({
             ],
           })
         }
-
-        Alert.alert(
-          LL.ScanningQRCodeScreen.invalidTitle(),
-          LL.ScanningQRCodeScreen.invalidContent({
-            found: data.toString(),
-          }),
-          [
-            {
-              text: LL.common.ok(),
-              onPress: () => setPending(false),
-            },
-          ],
-        )
+        if(destination.invalidReason=="InvoiceExpired"){
```

Comment:
instead of using the string `InvoiceExpired`, you should use the associated enum: `InvalidDestinationReason.InvoiceExpired`